README.md

```diff
@@ -26,6 +26,8 @@ Parse a HTTP `Cookie` header string and returning an object of all cookie name-v
 The `str` argument is the string representing a `Cookie` header value and `options` is an
 optional object containing additional parsing options.
 
+**NOTE:** This method does not parse the `set-cookie` header. It parses the `cookie` header. This means you cannot do `cookie.parse(cookie.serialize('name', 'value'))`.
```

Comment:
The example can be improved, as that code works fine.

serialize w/ no options essentially makes a valid `Cookie` string as it just sets the cookie name and value.
The real trouble (data corruption) is when folks use the options. Specifically the flag options like `HttpOnly, Secure, Partitioned` which aren't key value pairs like a cookie header.

I think we should give folks a little more info so they understand the failure state and risk.

Reply:
```suggestion
> ⚠️ **NOTE:** This method does not parse the `set-cookie` header. It parses the `cookie` header. This means you cannot safely parse a `set-cookie` header, or the output of `serialize`. 
>
> E.g. `cookie.parse(cookie.serialize('name', 'value', { httpOnly: true, secure: true, partitioned: true }))` will drop the `httpOnly, secure, partitioned` values entirely. Parsing `set-cookie` header values will result in unexpected results!
```